cmd/commands/ceph.go

```diff
@@ -33,9 +36,20 @@ var CephCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		logging.Info("running 'ceph' command with args: %v", args)
-		_, err := exec.RunCommandInOperatorPod(cmd.Context(), clientSets, cmd.Use, args, operatorNamespace, cephClusterNamespace, false)
-		if err != nil {
-			logging.Fatal(err)
+		if args[0] == "daemon" {
+			if len(args) > 2 {
+				cephLabel := strings.Replace(args[1], "-", ".", 1)
```

Comment:
Could you factor out lines 41-42 to a helper method and then add unit tests for it? Anytime there is string parsing, I recommend unit tests.

Reply:
Also can you add a comment that shows the expected input? It's helpful to see why we need to replace a dash.